### ISSUE TITLE
Failed to upload

### ISSUE THREAD

**Issue by buveur** [avocode/avocode#1559]
Every time I try to sync artboards it starts "preparing artboards and symbol" and gets stuck into an error.
-----
App v. `2.14.7`
Avocode UID: 59781
```
Error: ENOENT: no such file or directory, stat '/var/folders/lz/0gk1ctm12d7b86h84_35zph40000gn/T/com.madebysource.avocode/avocode-sync-1501227859254/7F0B8334-812C-4A69-8FF4-32A8253D0330/226E0338-8C28-4C80-BAAD-BA88722EB739.png'
```

**Comment by jakubzitny** [comment]
Hi @buveur, thanks for letting us know. This is a problem with our older Sketch plugin. We had some issues upgrading it automatically with the latest version, so please download the latest Avocode plugin from [here](https://upx.cz/iYd), double click to install it and let me know if it helps, please 🙂

**Comment by buveur** [comment]
Thanks @jakubzitny, it seems to work that way. But I had to manually remove older version of the plugin in order to upgrade it - not just the "double click to install" 🙂

**Comment by jakubzitny** [comment]
Allright, thank you for the info. I am glad it works for you now.

Can you please tell me what is the location of your plugins folder, please? Can you also run the following command in `Terminal` as well, so we can check if it's the same and install it to correct location next time?
```
/Applications/Sketch.app/Contents/Resources/sketchtool/bin/sketchtool show plugins; echo
```

**Comment by buveur** [comment]
The location is /Users/apple/Library/Application Support/com.bohemiancoding.sketch3/Plugins

It seems to be the same:
/Applications/Sketch.app/Contents/Resources/sketchtool/bin/sketchtool show plugins; echo
/Users/apple/Library/Application Support/com.bohemiancoding.sketch3/Plugins

**Comment by jakubzitny** [comment]
Thanks a lot @buveur, this is interesting, I have to inspect this further. Would you mind sharing your access rights to your plugins folder?

```
ls -l /Users/apple/Library/Application\ Support/com.bohemiancoding.sketch3/Plugins
ls -ls /Users/apple/Library/Application\ Support/com.bohemiancoding.sketch3/Plugins
```

**Comment by jakubzitny** [comment]
Hi there, this is fixed in latest Avocode `2.15.0`. You can download it from [avocode.com/download](https://avocode.com/download) or use auto-update to install it. Please give it a try and let us know if you have any more issues.
